src/System/PlayerHitPointData.cpp

```diff
@@ -0,0 +1,13 @@
+#include "System/PlayerHitPointData.h"
+
+PlayerHitPointData::PlayerHitPointData() {}
+
+void PlayerHitPointData::setKidsModeFlag(bool isKidsMode) {
+    mIsKidsMode = isKidsMode;
+}
+
+void PlayerHitPointData::init() {
+    mIsHaveMaxUpItem = false;
+    mIsForceNormalHealth = false;
+    mCurrentHit = !mIsKidsMode ? 3 : 6;
```

Comment:
Does it have to be "flipped" this way, or can it also be `mIsKidsMode ? 6 : 3`?

Reply:
This one does require that its flipped, `ne` vs `eq`. Possible that they used an if/else here to make more sense of the order but that's ugly.